packages/xo-web/src/xo-app/vm-import/esxi/esxi-import.js

```diff
@@ -0,0 +1,260 @@
+              <InputCol>
+                <input
+                  checked={hasCertificate}
+                  name='hasCertificate'
+                  onChange={this._toggleCertificateCheck}
+                  type='checkbox'
+                  value={hasCertificate}
+                />
+              </InputCol>
+            </Row>
+            <div className='form-group pull-right'>
+              <ActionButton btnStyle='primary' className='mr-1' handler={this._connect} icon='connect' type='submit'>
+                {_('serverConnect')}
+              </ActionButton>
+              <Button onClick={this._reset}>{_('formReset')}</Button>
+            </div>
+          </form>
+        ) : (
+          <form id='esxi-migrate-form'>
+            <Row>
+              <LabelCol>{_('vm')}</LabelCol>
+              <InputCol>
+                <Select
+                  disabled={isEmpty(vmsById)}
+                  onChange={this._onChangeVm}
+                  options={this._getSelectVmOptions()}
+                  required
+                  value={vm}
+                />
+              </InputCol>
+            </Row>
+            <Row>
+              <LabelCol>{_('vmImportToPool')}</LabelCol>
+              <InputCol>
+                <SelectPool onChange={this._onChangePool} required value={pool} />
+              </InputCol>
+            </Row>
+            <Row>
+              <LabelCol>{_('vmImportToSr')}</LabelCol>
+              <InputCol>
+                <SelectSr
+                  disabled={pool === undefined}
+                  onChange={this._onChangeSr}
+                  predicate={this._getSrPredicate()}
+                  required
+                  value={sr}
+                />
+              </InputCol>
+            </Row>
+            <Row>
+              <LabelCol>{_('network')}</LabelCol>
+              <InputCol>
+                <SelectNetwork
+                  disabled={pool === undefined}
+                  onChange={this._onChangeNetwork}
+                  predicate={this._getNetworkPredicate()}
+                  required
+                  value={network}
+                />
+              </InputCol>
+            </Row>
+            {vm !== undefined && (
+              <div>
+                <hr />
+                <h5>{_('vmsToImport', { nVms: 1 })}</h5>
+                <VmData data={vmsById[vm.value]} />
+              </div>
+            )}
+            <div className='form-group pull-right'>
+              <ActionButton
+                btnStyle='primary'
+                className='mr-1'
+                disabled={vm === undefined}
+                form='esxi-migrate-form'
+                handler={this._importVm}
+                icon='import'
+                redirectOnSuccess={getRedirectUrl}
+                type='submit'
+              >
+                {_('newImport')}
+              </ActionButton>
+              <Button onClick={this._reset}>{_('formReset')}</Button>
```

Comment:
We should probably also add a button to "disconnect" that would simply assign `false` to `isConnected`, what do you think?